src/RecognitionManager.js

```diff
@@ -26,14 +26,14 @@ export default class RecognitionManager {
   }
 
   setSpeechRecognition(SpeechRecognition) {
-    if (SpeechRecognition) {
+    if (SpeechRecognition && browserSupportsPolyfills()) {
```

Comment:
This logic is also run when _not_ using a polyfill. There are some older Chrome versions that do support Web Speech API but don't support MediaDevices. Those should still work without a polyfill - this line wrongly marks their browser support as false.

TODO: perhaps a flag is needed to identify whether a polyfill is being passed in or not.